### PR TITLE
 a new week

### PR DESCRIPTION
**Pull request by djarecka**

### CODE REVIEW

File: ME-MYSELF-I/Weekly-meetings.md [hunk 1]
```diff
@@ -1,6 +1,61 @@
...
+#### What did you achieve?
+
+* rerun all examples from nipype_tutorial and adding to datalad repo (it's very inefficient, still have to improve the workflow)
+* finished PR on networkx
+* run CWL tutorial 
+* solved an issue
+
+#### What did you struggle with?
+
+* I'm still not sure hot to efficiently create datalad repository from files from dropbox. Yarik corrected my files names, but I'm not sure what is the best way of using `git annex addurl` (more in my email).
```
Comment (satra) [inline ME-MYSELF-I/Weekly-meetings.md:32]: first something like this:

```python
def get_paths(root):
    paths = []
    values = dbx.files_list_folder(root)
    for entry in values.entries:
        if hasattr(entry, 'size'):
            paths.append(entry.path_lower)
        else:
            paths.extend(get_paths(entry.path_lower))
    return paths

foo = get_paths('/path/on/dropbox')
len(foo)

data = []
for path in foo:
    res = dbx.sharing_create_shared_link(path)
    data.append([path, res.url])

import json
with open('urls.json', 'wt') as fp:
    json.dump(data, fp, indent=2)
```

File: ME-MYSELF-I/Weekly-meetings.md [hunk 2]
```diff
@@ -1,6 +1,61 @@
...
+* debugging Alex workflow (and understanding how this should be done)
+* finish the datalad repo
+* continue running CWL examples
+
+
+
+#### Where do you need help from Satra?
+
+* gita annex/ datalad
+* possibly with traitlets
```
Comment (satra) [inline ME-MYSELF-I/Weekly-meetings.md:53]: i'll take a look at traitlets this week.

**Comment by satra** [comment]
lgtm

File: ME-MYSELF-I/Weekly-meetings.md [hunk 3]
```diff
@@ -1,6 +1,61 @@
 # Weekly Meetings
 
 * [15th September 2017](#date-15th-september-2017)
+* [24th September 2017](#date-15th-september-2017)
```
Comment (satra) [inline ME-MYSELF-I/Weekly-meetings.md]: wrong date.

File: ME-MYSELF-I/Weekly-meetings.md (hunk 1 above)
Comment (djarecka) [reply ME-MYSELF-I/Weekly-meetings.md:32]: Thanks, but I checked your code, and it looks like it does the same as my code, I'm creating the same list doing:

```
dbx = Dropbox(auth_code)

res = dbx.files_list_folder(path='/ds000114-nipype_tutorial_output', recursive=True)
cur = res.cursor
res_cd = dbx.files_list_folder_continue(cur)

with open('url_dj.txt', 'w') as file:
    for en in res_cd.entries:
        if hasattr(en, 'size'):
            file.write(en.path_lower + ", "  
                           + dbx.sharing_create_shared_link(en.path_lower).url.replace("www", "dl")      
                            + "\n")
```

Comment (djarecka) [reply ME-MYSELF-I/Weekly-meetings.md:32]: i'm just changing  `www` to `dl`

Comment (djarecka) [reply ME-MYSELF-I/Weekly-meetings.md:32]: Still don't work with `git annex` in a way I'm trying to use it: `git annex addurl --file <line.split(',')[0]> <ll.split(',')[1]>` 
gives me "Could not switch to '/ds000114-nipype_tutorial_output/': No such file or directory" if i don't create the directory

File: ME-MYSELF-I/Weekly-meetings.md (hunk 2 above)
Comment (djarecka) [reply ME-MYSELF-I/Weekly-meetings.md:53]: thank, I actually also haven't checked anything last week

File: ME-MYSELF-I/Weekly-meetings.md (hunk 1 above)
Comment (djarecka) [reply ME-MYSELF-I/Weekly-meetings.md:32]: ok, I was reading my files and found one more problem.... it might work now...

Comment (djarecka) [reply ME-MYSELF-I/Weekly-meetings.md:32]: Ok, so my problems came from the `entry.path_lower`. I didn't notice that it has always `/`, `git annex` was giving me only info " No such file or directory", so I thought that I have to create everything.

Everything works now and you can install from github: https://github.com/djarecka/datalad_repo_test

I can change names and I'm not sure if you want to connect it somehow to the main repo, or just keep it separately (so people don't download everything by running random commands).

**Pull request merged**